src/sonic-bgpcfgd/bgpcfgd/managers_bbr.py

```diff
@@ -45,26 +45,38 @@ def del_handler(self, key):
 
     def __init(self):
         """ Initialize BBRMgr. Extracted from constructor """
-        if not 'bgp' in self.constants:
-            log_err("BBRMgr::Disabled: 'bgp' key is not found in constants")
-            return
-        if 'bbr' in self.constants['bgp'] \
-                and 'enabled' in self.constants['bgp']['bbr'] \
-                and self.constants['bgp']['bbr']['enabled']:
+        # Check BGP_BBR table from config_db first
+        bbr_status_from_config_db = self.get_bbr_status_from_config_db()
```

Comment:
@Gfrom2016 could we validate the return value of status?

Reply:
Add validating return value in get_bbr_status_from_config_db()